tools/changelog/cmd/create.go

```diff
@@ -0,0 +1,215 @@
+// Copyright 2016-2022, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"time"
+
+	"github.com/goccy/go-yaml"
+	"github.com/gosimple/slug"
+	"github.com/pulumi/watchutil-rs/tools/changelog/changelog"
+	"github.com/spf13/cobra"
+
+	"github.com/erikgeiser/promptkit/selection"
+)
+
+var typ string
+var scope string
+var desc string
+var title string
+var force bool
+var issues []int
+var prs []int
+var outDir string
+var configFilename string
+
+// createCmd represents the create command
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create a changelog entry",
+	Long:  `Creates changelog entries in a project. Example:`,
+	RunE:  runCreate,
+}
+
+func runCreate(cmd *cobra.Command, args []string) error {
```

Comment:
Similar comment here as below. There's a long going on here. It might be worth running golang-ci-lint over the repo and noting the cyclomatic complexity of the largest functions. I suspect we may seek to drive that number down.